torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1330,6 +1348,30 @@ void ComputeConstant(Node* n, int opset_version) {
       ProcessUnchangeNode(n);
       break;
     }
+    case ::c10::onnx::Squeeze: {
+      if (ConstantValueMap::HasShape(n->input(0)->debugName())) {
+        auto input0_shape_size =
+            ConstantValueMap::GetShape(n->input(0)->debugName())
+                .value()
+                .sizes();
+        if (input0_shape_size.has_value()) {
+          auto input0_shape_value = input0_shape_size.value();
+          // For opset_versions > 13, for n->inputs().size() > 1,
```

Comment:
The if condition already tells us the detailed constraints. So we can just comment like this:
`// Other cases are handled via onnx shape inference.`